rotkehlchen/data_import/importers/rotki_trades.py

```diff
@@ -45,11 +41,7 @@ def _consume_rotki_trades(
         amount_bought = deserialize_asset_amount(csv_row['Buy Amount'])
         location = Location.deserialize(csv_row['Location'])
         trade = Trade(
-            timestamp=deserialize_timestamp_from_date(
-                date=csv_row['Timestamp'],
-                formatstr=timestamp_format,
-                location='Rotki generic trades importer',
-            ),
+            timestamp=deserialize_timestamp(timestamp=csv_row['Timestamp']),
             location=location,
             fee=fee,
             fee_currency=fee_currency,
```

Comment:
At line 65 I think **kwargs should not be passed since `_consume_rotki_trades` doesn't support any extra arguments anymore

Reply:
Bte I think might make sense to add a test where caller tries to pass `timestamp_format`. I guess in the current configuration due to kwargs it would fail

Reply:
> At line 65 I think **kwargs should not be passed since `_consume_rotki_trades` doesn't support any extra arguments anymore

Yeah, you're right. Thanks